tests/aftu/test_compare_graphs.py

```diff
@@ -0,0 +1,174 @@
+"""Compare graphs generated by vLLM-Spyre vs AFTU
+
+Run `python -m pytest tests/aftu/test_compare_graphs.py`.
+"""
+
+import os
+import sys
+import tempfile
+
+import pytest
+from graph_compare_utils import (collect_graph_files, compare_graphs,
+                                 get_model_path,
+                                 run_inference_py_and_get_graphs)
+from spyre_util import (generate_spyre_vllm_output, get_chicken_soup_prompts,
+                        get_spyre_model_list)
+from vllm import SamplingParams
+
+
+@pytest.mark.spyre
+@pytest.mark.aftu
+@pytest.mark.parametrize("model", get_spyre_model_list())
+@pytest.mark.parametrize("max_num_seqs", [2, 4],
+                         ids=lambda val: f"max_num_seqs({val})")
+def test_compare_graphs_cb(
+    model: str,
+    max_num_seqs: int,
+    monkeypatch: pytest.MonkeyPatch,
+    runtime_xfail,
+):
+    """Test that the spyre worker correctly outputs
+    continuous batches of requests by comparing to HF"""
+
+    if max_num_seqs > 2:
+        runtime_xfail("CB failures expected for batch size > 2")
```

Comment:
This should have since been fixed, let's make sure the test passes with --max-num-seqs 4

Reply:
Looks like it passes for me on a dev pod 👍 

Should be good to remove this check